### PR TITLE
1.20.3/4 Implementation

### PR DESCRIPTION
**Pull request by milutinke**
Do not merge until it's fully tested

### CODE REVIEW

File: MinecraftClient/Protocol/Handlers/Protocol18.cs [hunk 1]
```diff
@@ -361,8 +365,8 @@ internal void PacketReader(object? o)
         /// <returns>TRUE if the packet was processed, FALSE if ignored or unknown</returns>
         internal bool HandlePacket(int packetId, Queue<byte> packetData)
         {
-            try
-            {
+            //try
+            //{
```
Comment (ReinforceZwei) [inline MinecraftClient/Protocol/Handlers/Protocol18.cs:369]: I forgot to uncomment the try-catch here. Remember to uncomment it before release.

File: MinecraftClient/Protocol/Handlers/Protocol18.cs [hunk 2]
```diff
@@ -460,26 +449,73 @@ internal bool HandlePacket(int packetId, Queue<byte> packetData)
...
+
+            //    throw new System.IO.InvalidDataException(
+            //        string.Format(Translations.exception_packet_process,
+            //            packetPalette.GetIncomingTypeById(packetId),
+            //            packetId,
+            //            protocolVersion,
+            //            currentState == CurrentState.Login,
+            //            innerException.GetType()),
+            //        innerException);
+            //}
```
Comment (ReinforceZwei) [inline MinecraftClient/Protocol/Handlers/Protocol18.cs:468]: I forgot to uncomment the try-catch here. Remember to uncomment it before release.

**Review by ReinforceZwei — CHANGES_REQUESTED** [review]

File: MinecraftClient/Protocol/Handlers/Protocol18.cs (hunk 1 above)
Comment (milutinke) [reply MinecraftClient/Protocol/Handlers/Protocol18.cs]: Done

**Review by ReinforceZwei — APPROVED** [review]

**Comment by milutinke** [comment]
Since no one has made any new bug reports in a while I will merge it.

**Pull request merged**